docs/site.json

```diff
@@ -28,7 +28,7 @@
   ],
   "pluginsContext" : {
     "filterTags" : {
-      "tags": ["environment--ug", "environment--dg"]
+      "tags": ["environment--combined"]
```

Comment:
This is really a minor nit 😅, but I personally think that `--combined` is not the best wording for this... Maybe something like `environment--guide` makes more sense? Since this environment is purely for the user guide and the dev guide. 

I think `--combined` might have a connotation of being an overarching environment for all the pages (not just the user guide & dev guide), what do you think?

Reply:
I think `--combined` accurate as the docs served with `site.json` is the combination of the ug and dg. What do the rest think?

Reply:
I think @lhw-1's concern is reasonable, but "combined" in this instance as a reference to the whole site (with both UG and DG) seems fine to me.